Source/WebCore/css/parser/CSSTokenizer.cpp

```diff
@@ -682,7 +689,7 @@ CSSParserToken CSSTokenizer::consumeURLToken()
             return CSSParserToken(UrlToken, registerString(result.toString()));
 
         if (isASCIIWhitespace(cc)) {
-            m_input.advanceUntilNonWhitespace();
+            m_input.advanceUntilNewlineOrNonWhitespace();
```

Comment:
This now stops at a newline, which presumably fixes a bug? Do we have a test case covering that change in behavior?

Reply:
Reviewed the CSS Spec[1] and added 2 additional tests in `CSSTokenizerTests.cpp` to validate this. Fixed.

Interestingly enough, I was unable to find any behavior differences (both in any API or WebPlatformTests) nor on any websites when running a local build of `advancingUntilNonWhitespace` vs `advancingUntilNewlineOrNonNewline`.

I believe this is due to how we handle tokens after the first `(` of a URL.

[1] https://drafts.csswg.org/css-syntax/#consume-ident-like-token

Reply:
I guess then that we should not change this code since it doesn't need to be changed.